app/build.gradle.kts

```diff
@@ -161,10 +161,12 @@ dependencies {
     }
     implementation(libs.okhttp)
     implementation(libs.okhttp.logging.interceptor)
+    implementation("com.google.code.gson:gson:2.10.1")
```

Comment:
Please define the version and module in the `libs.versions.toml`, similar to the imports of the `mockk` and `okhttp` libraries.